FrontEnd/Hopsy/src/screens/ProfileScreen.js

```diff
@@ -29,19 +29,19 @@ export default class ProfileScreen extends React.Component {
           username: GLOBAL.user,
         })
     })  
-    .then((response) => response.text())
-      .then((responseJson) => {
-        this.setState({
-          fullname: responseJson,
-        }, function() {
-        })
-      })
+    .then(response => {
```

Comment:
add comments so that it is easier for others to follow, and where the other methods could be used

Reply:
I agree

Reply:
Explanatory comment added in 
https://github.com/nauticalmonkey/CSC308/commit/9273d5ca9b47ebdbb5da410f31ace01fe27bcb11